samples/bluetooth/bap_broadcast_sink/src/main.c

```diff
@@ -91,12 +91,17 @@ static const struct bt_audio_codec_cap codec_cap = BT_AUDIO_CODEC_CAP_LC3(
 	CONFIG_MAX_CODEC_FRAMES_PER_SDU,
 	(BT_AUDIO_CONTEXT_TYPE_CONVERSATIONAL | BT_AUDIO_CONTEXT_TYPE_MEDIA));
 
-/* Create a mask for the maximum BIS we can sync to using the number of streams
- * we have. Bit 0 is BIS index 1.
- */
-static const uint32_t bis_index_mask = BIT_MASK(CONFIG_BT_BAP_BROADCAST_SNK_STREAM_COUNT);
+struct bis_index_data {
```

Comment:
This struct contains more than just bis_index data, and we should consider another name for the struct (and the variable). 

Maybe something along the lines of `base_sync_info`?

Reply:
We could also consider moving some of the other global static variables in here, as they sort of serve the same purpose